rotkehlchen/tests/utils/dataimport.py

```diff
@@ -1556,7 +1558,45 @@ def assert_binance_import_results(rotki: Rotkehlchen):
             link='',
             notes='Imported from binance CSV file. Binance operation: Buy / Sell',
         ),
+        Trade(
+            timestamp=Timestamp(1690262284),
+            location=Location.BINANCE,
+            base_asset=A_USDT,
+            quote_asset=A_USDC,
+            trade_type=TradeType.BUY,
+            amount=FVal('2641.26410000'),
+            rate=Price(FVal('1.0001')),
+            link='',
+            notes='Imported from binance CSV file. Binance operation: Buy / Sell',
+        ),
+        Trade(
+            timestamp=Timestamp(1690570138),
+            location=Location.BINANCE,
+            base_asset=A_USDT,
+            quote_asset=A_SOL,
+            trade_type=TradeType.BUY,
+            amount=FVal('26.82490000'),
+            rate=Price(FVal('25.07')),
+            fee=FVal(-0.00008316),
```

Comment:
fee should be a positve amount. We always work with positve amounts

Reply:
I think it's different for trades:
https://github.com/rotki/rotki/blob/03f738d44412f7429105f1edf24c5a4cc315e70b/rotkehlchen/tests/utils/dataimport.py#L1515

Reply:
Why you mention it. Checking the logic when processed is assumed to be positive as seen here https://github.com/rotki/rotki/blob/e064d8db4898c7e7b681ea360c8d73607e528c24/rotkehlchen/exchanges/data_structures.py#L441

Reply:
If I search the current codebase for this: `Fee(FVal('-` there're valid results [here](https://github.com/search?q=repo%3Arotki%2Frotki+Fee%28FVal%28%27-&type=code).
So it's probably a bug that has been merged into develop and I should fix everything with mine?

Reply:
Seems to be binance import tests again?

There is no such thing as a negative fee. If there is a negative fee. It does not make sense. Something is wrong if there is one.